robotPositioner/Makefile

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
There is a 'base' makefile in  ``common/flags.mk`` you can include to save some typing here -- see ``examples/see3cam/Makefile`` for an example